### PR TITLE
feat(cli): add externalsType receive

### PR DESCRIPTION
**Pull request by myNameIsDu**
- [x] add externalsType receive
- [ ] add refresh-plugin cofig receive

### CODE REVIEW

File: packages/cli/ak-webpack-config/index.js [hunk 1]
```diff
@@ -87,6 +89,7 @@ const generateConfig = config => {
         resolve: configResolve,
         resolveLoader: configResolveLoader,
         externals: configExternals,
+        externalsType: configExternalTye,
```
Comment (bigbigDreamer) [inline packages/cli/ak-webpack-config/index.js:92]:
```suggestion
        externalsType: configExternalsType,
```

File: packages/cli/ak-webpack-config/index.js [hunk 2]
```diff
@@ -158,6 +161,9 @@ const generateConfig = config => {
     if (configExternals) {
         webpackConfig.externals = configExternals;
     }
+    if (configExternalTye) {
+        webpackConfig.externalsType = configExternalTye;
```
Comment (bigbigDreamer) [inline packages/cli/ak-webpack-config/index.js:165]:
```suggestion
        webpackConfig.externalsType = configExternalsType;
```

**Review by bigbigDreamer — COMMENTED** [review]
good job

File: packages/cli/ak-webpack-config/index.js [hunk 3]
```diff
@@ -158,6 +161,9 @@ const generateConfig = config => {
     if (configExternals) {
         webpackConfig.externals = configExternals;
     }
+    if (configExternalTye) {
```
Comment (bigbigDreamer) [inline packages/cli/ak-webpack-config/index.js:164]:
```suggestion
    if (configExternalsType) {
```

**Review by bigbigDreamer — COMMENTED** [review]
bad boy , please

File: packages/cli/ak-webpack-config/index.js (hunk 3 above)
Comment (myNameIsDu) [reply packages/cli/ak-webpack-config/index.js:164]: :-)

**Review by bigbigDreamer — APPROVED** [review]
good job

**Pull request merged**